release/air_tests/air_benchmarks/mlperf-train/resnet50_ray_air.py

```diff
@@ -530,17 +548,40 @@ def append_to_test_output_json(path, metrics):
 
     trainer = TensorflowTrainer(
         train_loop_for_worker,
-        scaling_config=ScalingConfig(num_workers=1, use_gpu=args.use_gpu),
+        scaling_config=ScalingConfig(
+            num_workers=1,
+            use_gpu=args.use_gpu,
+            trainer_resources={"CPU": args.trainer_resources_cpu},
+        ),
         datasets=datasets,
         preprocessor=preprocessor,
         train_loop_config=train_loop_config,
     )
+
+    tuner = None
+    if args.tune_trials > 0:
+        tuner = Tuner(
+            trainer,
+            param_space={
+                "train_loop_config": {
+                    "random_var": tune.grid_search(list(range(1, args.tune_trials + 1)))
```

Comment:
nit: I think you can remove `list`, since the API says it takes an iterable https://docs.ray.io/en/latest/tune/api_docs/search_space.html#grid-search-api